sys/mips/include/cheri.h

```diff
@@ -141,6 +141,8 @@ void	cheriabi_newthread_init(struct thread *td);
 void	hybridabi_exec_setregs(struct thread *td, unsigned long entry_addr);
 void	hybridabi_newthread_setregs(struct thread *td,
 	    unsigned long entry_addr);
+void * __capability hybridabi_user_ddc(void);
```

Comment:
This might eventually take a `struct thread *td` if we end up reducing DDC for library compartmentalization in hybrid programs, but probably point in adding it now.